editor.planx.uk/src/pages/FlowEditor/components/Sidebar/PreviewBrowser/index.tsx

```diff
@@ -0,0 +1,51 @@
+
+import Button from "@mui/material/Button";
+import { createTheme, styled, ThemeProvider, useTheme } from "@mui/material/styles";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React from "react";
+import Reset from "ui/icons/Reset";
+
+import Questions from "../../../../Preview/Questions";
+
+const ResetToggle = styled(Button)(({ theme }) => ({
+  position: "absolute",
+  top: 0,
+  right: theme.spacing(3),
+  padding: theme.spacing(1, 1, 1, 0),
+  textDecorationStyle: "solid",
+  color: theme.palette.text.primary,
+}));
+
+export const PreviewBrowser: React.FC = () => {
+  const [resetPreview] = useStore((state) => [state.resetPreview]);
+
+  const theme = useTheme();
+  const mobileTheme = createTheme({
+    ...theme,
+    breakpoints: {
+      values: {
+        xs: 0,
+        // Force mobile breakpoints as sidebar is pinned at 500px
+        sm: 9999,
+        md: 9999,
+        lg: 9999,
+        xl: 9999,
+        formWrap: 9999,
+        contentWrap: 9999,
+      },
+    },
+  });
+
+  return (
+    <ThemeProvider theme={mobileTheme}>
```

Comment:
A more robust solution here is probably a same-origin iframe - this could also solve our issues of the preview browser not going past the currently available nodes.

As it's same-origin, we could read and write the same local storage (for breadcrumbs etc).

A few good ways to solve this one!